### PR TITLE
Added flags for everything

### PR DESCRIPTION
**Pull request by WnndGws**
Love click so far

### CODE REVIEW

**Comment by WnndGws** [comment]
See how font needs two arguements, would it be better to do something like

`def quote_font(font, fontsize):` instead?

**Comment by WnndGws** [comment]
I agree. Json is the lovescock way to go

File: Python/wallpaperMaker/wallpaperMaker.py [hunk 1]
```diff
@@ -42,14 +76,24 @@
     help="Font size [DEFAULT: 50]"
 )
 
-def change_wallpaper(wallpaper_dir, quote_file, font, font_size):
+@click.option(
+    '--bing',
```
Comment (ammgws) [inline Python/wallpaperMaker/wallpaperMaker.py]: Does this need a default value?

**Comment by WnndGws** [comment]
@ammgws No, just havent gotten around to it yet.

Seems like next step would be for me to understand "Nesting Commands" for click

**Pull request merged**